auth/auth/driver/driver.py

```diff
@@ -307,6 +307,7 @@ def __init__(self, k8s_client, name=None):
         self.name = name
 
     async def create(self, name):
+        assert name != 'default'
```

Comment:
What is your reason for not making this stronger? I don't think you want to try and create and delete the namespace you are currently running in. I was thinking `assert name not in ('default', HAIL_DEFAULT_NAMESPACE)`